src/modules/people-list/people-list-item.tsx

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import Swipeable from 'react-native-swipeable';
+import {IProps} from './people-list-item.type';
+import Styled from 'styled-components/native';
+
+import {minorScale} from '../scales';
+import Text from '../text/text';
+import Avatar from '../avatar/avatar';
+import Checkbox from '../checkbox/checkbox';
+import Heading from '../heading/heading';
+
+// Wrapper
+const StyleWrapper = Styled.View`
+  flex-direction: row;
+  padding-bottom: ${minorScale(4)}px;
+  padding-top: ${minorScale(4)}px;
+  background-color: ${({theme}) => {
+    return theme.colors.white;
+  }};
+`;
+
+const StyleProfilePicCheckboxWrapper = Styled.View`
+  flex-direction: row; 
+  justify-content: center
+`;
+
+const StyleCheckboxWrapper = Styled.View`
+  margin-top: ${minorScale(2)}px;
+  margin-right: ${minorScale(2)}px;
+`;
+
+const StyleUserDetailsWrapper = Styled.View<IProps>`
+  padding-left: ${minorScale(2)}px;
+  width: ${props => (props.isShowCheckbox === true ? '69%' : '82%')}};
+
+`;
+
+const StyleUserNameIdWrapper = Styled.View`
+  flex-direction: row; 
+  justify-content: space-between
+`;
+
+const StyleOtherDetailWrapper = Styled.View`
+  width:100%
+`;
+
+const StyleHeadingUserNameWrapper = Styled(Heading)`
+  font-size: ${minorScale(3.6)}px;
+  width: 80%;
+`;
+
+const StyleUserIdTextWrapper = Styled(Text)`
+  font-size: ${minorScale(2.8)}px;
+  color: ${({theme}) => {
+    return theme.colors.grayFour;
+  }};
```

Comment:
Please try to avoid manually colouring or changing font sizes for the text component. There are props like `variant`, `color` etc. on the Text component to handle these.

Reply:
Done